lib/Parser/Scan.cpp

```diff
@@ -587,7 +587,7 @@ IdentPtr Scanner<EncodingPolicy>::PidOfIdentiferAt(EncodedCharPtr p, EncodedChar
 }
 
 template <typename EncodingPolicy>
-typename Scanner<EncodingPolicy>::EncodedCharPtr Scanner<EncodingPolicy>::FScanNumber(EncodedCharPtr p, double *pdbl, bool& likelyInt)
+typename Scanner<EncodingPolicy>::EncodedCharPtr Scanner<EncodingPolicy>::FScanNumber(EncodedCharPtr p, double *pdbl, bool& likelyInt, size_t saveMultiUnits)
```

Comment:
Could you please rename this to `savedMultiUnits`? On my first reading I assumed it was a Boolean (GitHub code review interface didn't help by pushing the argument out of view)

Reply:
Sure, that makes sense. We are not overly consistent with the naming of these save variables, anyway.